model_to_omics_data_config.nocomments.json

```diff
@@ -0,0 +1,173 @@
+{
+    "table_filters": {
+        "omics_sequencing": {
+            "exclusion_filters": {
+                "columns": ["blacklist"],
+                "blacklist": [1],
+                "filter_mode": "all"
+            },
+            "inclusion_filters": {
+                "columns": ["expected_type"],
+                "expected_type": ["rna", "wgs", "wes"],
+                "filter_mode": "any"
+            }
+        },
+        "omics_profile": {
+            "supercede_filter": {
+                "columns": ["internal_release_date", "consortium_release_date", "public_release_date"],
+                "internal_release_date": ["None"],
+                "consortium_release_date": ["None"],
+                "public_release_date": ["None"],
+                "filter_mode": "any"
+            },
+            "exclusion_filters": {
+                "columns": ["blacklist_omics"],
+                "blacklist_omics": [1],
+                "filter_mode": "all"
+            },
+            "inclusion_filters": {
+                "columns": ["datatype"],
+                "datatype": ["rna", "wgs", "wes"],
+                "filter_mode": "any"
+            },
+            "merge_parameters" : {
+                "merge_with" : ["omics_sequencing"],
+                "primary" : ["profile_id"],
+                "foreign" : ["profile_id"]
+            }
+        },
+        "model_condition": {
+            "exclusion_filters": {
+                "columns": ["expansion_team"],
+                "expansion_team": ["PRISM"],
+                "filter_mode": "all"
+            },
+            "inclusion_filters": {
+                "columns": ["drug"],
+                "drug": ["None", "DMSO"],
```

Comment:
From my conversation with Sam in 24Q4, profiles that were drug-treated can never be default, but they can still be released. Has that changed?